flow-model-generator/modules/flow-model-generator-core/src/main/java/io/ballerina/flowmodelgenerator/core/DataMapManager.java

```diff
@@ -681,6 +687,38 @@ private void genSource(ExpressionNode expr, String[] names, int idx, StringBuild
         }
     }
 
+    private void setImportStatements(Map<String, String> importStatements, List<TextEdit> textEdits) {
+        if (importStatements == null) {
+            return;
+        }
+
+        ModulePartNode modulePartNode = document.syntaxTree().rootNode();
+        List<ImportDeclarationNode> importDeclNodes = modulePartNode.imports().stream().toList();
+
+        for (String importStatement : importStatements.values()) {
+            Module module = document.module();
+            ModuleDescriptor descriptor = module.descriptor();
```

Comment:
We can combine them and remove `module` variable.

```suggestion
            ModuleDescriptor descriptor = document.module().descriptor();
```

Reply:
Fixed